frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/atoms/Table.js

```
@@ -159,156 +155,157 @@ const Table = ({
+                        {showAutoSerialNo && typeof showAutoSerialNo == "string" ? t(showAutoSerialNo) : t("TB_SNO")}
+                      </th>
+                    )}
+                    {headerGroup.headers.map((column) => (
+                      <th
+                        {...column.getHeaderProps(column.getSortByToggleProps())}
+                        style={
+                          column?.id === "selection"
+                            ? { minWidth: "100px" }
+                            : { verticalAlign: "top", textAlign: `${column?.headerAlign ? column?.headerAlign : "left"}` }
+                        }
+                      >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)